services/minespace-web/src/components/common/DocumentTable.tsx

```diff
@@ -7,34 +7,35 @@ import {
   uploadedByColumn,
 } from "./DocumentColumns";
 import {
-  renderTextColumn,
-  renderActionsColumn,
   ITableAction,
+  renderActionsColumn,
+  renderTextColumn,
 } from "@mds/common/components/common/CoreTableCommonColumns";
 import { some } from "lodash";
 import { closeModal, openModal } from "@mds/common/redux/actions/modalActions";
+import DocumentCompression from "@mds/common/components/documents/DocumentCompression";
 import { archiveMineDocuments } from "@mds/common/redux/actionCreators/mineActionCreator";
-import { connect } from "react-redux";
-import { bindActionCreators } from "redux";
+import { useDispatch, useSelector } from "react-redux";
 import { modalConfig } from "@/components/modalContent/config";
 import { Feature } from "@mds/common";
 import { SizeType } from "antd/lib/config-provider/SizeContext";
-import { ColumnType, ColumnsType } from "antd/es/table";
-import DeleteOutlined from "@ant-design/icons/DeleteOutlined";
-import DownloadOutlined from "@ant-design/icons/DownloadOutlined";
-import FileOutlined from "@ant-design/icons/FileOutlined";
-import InboxOutlined from "@ant-design/icons/InboxOutlined";
-import SyncOutlined from "@ant-design/icons/SyncOutlined";
+import { ColumnsType } from "antd/es/table";
+import { FileOperations, MineDocument } from "@mds/common/models/documents/document";
+import {
+  DeleteOutlined,
+  DownloadOutlined,
+  DownOutlined,
+  FileOutlined,
+  InboxOutlined,
+  SyncOutlined,
+} from "@ant-design/icons";
```

Comment:
I think we want to keep the antd icon imports how they were, it was one of Simen's changes for decreased bundle sizes.

Reply:
Oh oops!  Was copying some stuff from the other DocumentTable.  Good catch!